test/testdrive/mzcompose.yml

```diff
@@ -8,29 +8,95 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  # Run tests with no support for s3
+  local:
+    steps:
+      - step: workflow
+        workflow: start-deps
+      - step: run
+        service: testdrive
+
+  # Run tests with s3 stubbed out, no credentials needed
+  local-aws:
+    env:
+      AWS_REGION: localstack
+      AWS_ENDPOINT: http://localstack:4566
```

Comment:
These were previously the defaults, so if we leave the CLI parsing the way it was there should be no need to set these.

Reply:
setting these explicitly like this will override whatever the user has in their environment, guaranteeing that it works the way that it's described, otherwise we are possibly allowing things from the environment to corrupt the run. I'd rather only have one workflow that allows passing these env vars through. Plausibly that is the `testdrive` workflow. I'll put up a proposal.

Reply:
oh actually the endpoint is not the default, here the host is `localstack` and the default is `localhost`

Reply:
Oh, yeah, I see, so this would have to stay `AWS_ENDPOINT=http://localhost:4566`.

Reply:
Oh, I just realized that the endpoint is pseudo-default: the parameter defaults to `None` but then if we get into the alternate branch we unwrap a default. That seems... subtle. I'd rather just entirely remove the pseudo-default behavior and give folks a clear error message. 

I'm happy to switch over the opt-parsing to be strict so that their either required to be consistent or not at all, but I'm not sure how valuable that work is for testdrive right now.

Reply:
I've added a proposed commit that implements this.